frontend/src/components/Navbar/index.jsx

```diff
@@ -17,42 +17,72 @@ export const Navbar = () => {
         setError(error.message);
       });
   }, []);
+  const toggleMobileNav = () => {
+    document.getElementById('mobileNav').classList.toggle('hidden');
```

Comment:
It's not a good practice to use getElementById .. Avoid manipulating DOM directly 
React have all the tools you need to achieve the same result and logic